index.js

```diff
@@ -344,22 +350,16 @@ let rcl = {
    * this folder will be overwritten after the execution.
    * @return {Promise<undefined>} A Promise.
    */
-  regenerateAll() {
+  async regenerateAll() {
     // This will trigger to regenerate all the JS structs used for messages and services,
     // to overwrite the existing ones although they have been created.
-    debug('Begin regeneration of JavaScript code from ROS IDL files.');
-    return new Promise((resolve, reject) => {
-      generator
-        .generateAll(true)
-        .then(() => {
-          tsdGenerator.generateAll(); // create interfaces.d.ts
-          debug('Finish regeneration.');
-          resolve();
-        })
-        .catch((e) => {
-          reject(e);
-        });
-    });
+    debug('Begin regeneration of JavaScript code from ROS IDL files...');
+
+    # generate the messages and type declarations, which must be done in sequence
```

Comment:
I think this line causes the failure on ci :)

Reply:
Oh WTF, too much Python ... :sweat_smile: